### DISCUSSION TITLE
JSCAD vs JSModeler

### DISCUSSION THREAD

**Discussion by Daubsy** [jscad/OpenJSCAD.org#1188] [Q&A]
Hi All! I am starting a project (creating an assembly for 3D printing) and trying to decide on a library to use. I am considering JSCAD or JSModeler, [https://github.com/kovacsv/JSModeler](url). I am a novice programmer, so I would appreciate any help with identifying key differences between the two libraries, as well as weaknesses and strengths. Thank you! :)

**Comment by hrgdavor** [comment]
If you are just starting a project and choosing a library in this case it is rather clear of the two mentioned here, one is larger and actively developed, the other has last update 2 years ago.

**Reply by Daubsy** [reply]
Yes I did notice that! It does have similar functionality to jscad, and pretty good documentation, and I like that it works with three.js. However, since I don't know much about the underlying architecture, I am not sure if it can produce manifold meshes for 3D printing. I am leaning towards jscad, though I do wish connectors were still supported.

**Reply by hrgdavor** [reply]
There is already few prototypes of jscad that work with threjs. So the support for threjs will be coming.
